A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -1998,12 +1998,18 @@ class FMCMainDisplay extends BaseAirliners {
 
     setPerfApprVApp(s) {
         if (s === FMCMainDisplay.clrValue) {
-            this.vApp = NaN;
-        }
-        const value = parseFloat(s);
-        if (isFinite(value) && value > 0) {
-            this.vApp = value;
-            return true;
+            if (isFinite(this.vApp)) {
+                this.vApp = NaN;
+                return true;
+            }
+        } else {
+            const value = parseFloat(s);
```

Comment:
I am just seeing this we should use parseInt instead! (or detect a decimal point and throw "not allowed")

Reply:
The more you look at the existing code... I will fix ;).